apps/site/pages/components/organisms/out-of-stock.mdx

```diff
@@ -0,0 +1,202 @@
+---
+description: This component is used to announce to users that the product is out of stock.
+sidebar_custom_props:
+  image: https://vtexhelp.vtexassets.com/assets/docs/src/OutOfStock___5863b0eb4a4a8ce9a98da5c760461e61.png
+---
+
+import path from 'path'
+import { useSSG } from 'nextra/ssg'
+import { Tab, Tabs } from 'nextra-theme-docs'
+import PropsSection from 'site/components/PropsSection'
+import { TokenTable, TokenRow, TokenDivider } from 'site/components/Tokens'
+import { OverviewSection } from 'site/components/OverviewSection'
+import { OutOfStock } from '@faststore/ui'
+import { getComponentPropsFrom } from 'site/components/utilities/propsSection'
+
+export const getStaticProps = () => {
+  const outOfStockPath = path.resolve(__filename)
+  const components = ['OutOfStock.tsx']
+  const [outOfStockProps] = getComponentPropsFrom(outOfStockPath, components)
+  return {
+    props: {
+      // We add an `ssg` field to the page props,
+      // which will be provided to the Nextra `useSSG` hook.
+      ssg: {
+        outOfStockProps,
+      },
+    },
+  }
+}
+
+export const OutOfStockPropsSection = ({ component }) => {
+  // Get the data from SSG, and render it as a component.
+  const { outOfStockProps } = useSSG()
+  return {
+    OutOfStock: <PropsSection propsList={outOfStockProps} />,
+  }[component]
+}
+
+<header>
+
+# Out of Stock
+
+This component is used to announce to users that the product is out of stock. It also asks for their contact to let they know when the product arrives.
+
+</header>
+
+## Import
+
+Import the component from [@faststore/ui](/../components)
+
+```tsx
+import { OutOfStock } from '@faststore/ui'
+```
+
+Import Styles
+
+```tsx
+import '@faststore/ui/src/components/organisms/OutOfStock/styles.scss'
+```
+
+---
+
+## Usage
+
+<Tabs items={['Example', 'Code']} defaultIndex="0">
+  <Tab>
+    <OverviewSection containerStyle={{display: 'block'}}>
+      <OutOfStock
+        title="Out of Stock"
+        subtitle="Notify me when available"
+        inputLabel="Email"
+        buttonLabel="Notify Me"
+        onSubmit={()=>{}}
+        disabled={false}
+      />
+    </OverviewSection>
+  </Tab>
+  <Tab>
+    ```tsx
+    <OutOfStock
+        title="Out of Stock"
+        subtitle="Notify me when available"
+        inputLabel="Email"
+        buttonLabel="Notify Me"
+        onSubmit={handleSubmit}
+        disabled={isLoading}
+      />
+    ```
+
+  </Tab>
+</Tabs>
+
+---
+
+## Props
+
+### Out of Stock
+
+<OutOfStockPropsSection component="OutOfStock" />
+
+### Out of Stock Title
+
+export const propsOutOfStockTitle = [
```

Comment:
Have you ever tried changing the type from `OutOfStockTitleProps` for the interface? In my previous tests, types don't work as expected :(

Reply:
Lemme try! :)